cocos/physics-2d/framework/physics-system.ts

```diff
@@ -354,6 +357,7 @@ director.once(Director.EVENT_INIT, () => {
 
 function initPhysicsSystem () {
     if (!PhysicsSystem2D.PHYSICS_NONE && !EDITOR) {
-        director.registerSystem(PhysicsSystem2D.ID, PhysicsSystem2D.instance, System.Priority.LOW);
+        const sys = PhysicsSystem2D.instance;
+        director.registerSystem(PhysicsSystem2D.ID, sys, sys.priority);
```

Comment:
The default priority doesn't necessarily be LOW priority, please keep using `System.Priority.LOW`

Reply:
emmm, alright, because i read the source code from 3D `PhysicsSystem`, it uses this way, so i modified it.